app/src/main/java/com/conupods/IndoorMaps/View/PathOverlay.java

```diff
@@ -0,0 +1,53 @@
+package com.conupods.IndoorMaps.View;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.Polyline;
+import com.google.android.gms.maps.model.PolylineOptions;
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import android.graphics.Color;
+
+public class PathOverlay {
+
+    private GoogleMap mMap;
+
+    LatLng mNWHall = new LatLng(45.497161, -73.579554);
+    LatLng mSWHall = new LatLng(45.496827, -73.578850);
+    LatLng mSEHall = new LatLng(45.497370, -73.578337);
+    LatLng mNEHall = new LatLng(45.497710, -73.579035);
```

Comment:
I would suggest, thinking about a more general of approaching this problem, which would take advantage of dynamically creating such variables given the metadata. createPolyLine() only works for one hardcoded building right now.

Reply:
ya i'm looking into that, the other places where latlngs are stored aren't providing all the information i'll need so i may make something to store all those coordinates

Reply:
I can help you with this once my PR is pushed. There will be an asset file where you can hardcode these values and call in when you need them! I'm already calling all building metadata from a json file which also holds latlng for every conco building. So maybe you would need one for whatever this is.